### PR TITLE
fix(scrapers): fix /esami and /prof commands scrapers and minor stuff

### PR DESCRIPTION
**Pull request by scraccio**

### CODE REVIEW

File: module/callback_handlers.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 # Needed to correctly run functions using globals()
 from module.utils.multi_lang_utils import get_on_demand_text
 
+from module.commands.esami import *  # pylint: disable=wildcard-import,unused-wildcard-import
+from module.commands.lezioni import *  # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.help import * # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.drive_contribute import * # pylint: disable=wildcard-import,unused-wildcard-import
 from module.commands.aulario import * # pylint: disable=wildcard-import,unused-wildcard-import
```
Comment (Helias) [inline module/callback_handlers.py:16]: you should not use `from something import *`

File: module/commands/professori.py [hunk 2]
```diff
@@ -32,10 +32,10 @@ def prof(update: Update, context: CallbackContext) -> None:
 
     if len(professors) == 1:
         # fetch the "ID Foto" property from the resulting markdown
-        id_photo = message_text.split('\n')[-3].split(':')[1].replace('*', "")[1:]
+        id_photo = message_text.split('\n')[-3].split(':')[1].replace('*', "").replace('\\', "")[1:]
         # only sends a photo if it exists and if the result is only one
         if id_photo != "Non presente":
-            photo_url = "http://web.dmi.unict.it/foto_docenti/" + id_photo + ".jpg"
+            photo_url = "http://web.dmi.unict.it" + id_photo
```
Comment (Helias) [inline module/commands/professori.py:38]: no `/` after `http://web.dmi.unict.it`?

Comment (scraccio) [reply module/commands/professori.py:38]: The slash is already embedded in the src of the img tag

File: module/data/exam.py [hunk 3]
```diff
@@ -122,63 +129,83 @@ def scrape(cls, year_exams: str, delete: bool = False):
...
+                        # vuol dire che nelle sessioni precedenti non aveva appelli (oppure è la prima sessione)
+                        if not flag:
+                            if course == "l-31":
+                                course_name = "Informatica Triennale"
+                            elif course == "lm-18":
+                                course_name = "Informatica Magistrale"
+                            elif course == "l-35":
+                                course_name = "Matematica Triennale"
+                            elif course == "lm-40":
+                                course_name = "Matematica Magistrale"
```
Comment (Helias) [inline module/data/exam.py:177]: you could wrap this into a `dict`

File: module/callback_handlers.py (hunk 1 above)
Comment (scraccio) [reply module/callback_handlers.py:16]: Okay, I'll try to specify which method is actually needed

File: module/data/exam.py [hunk 4]
```diff
@@ -122,63 +129,83 @@ def scrape(cls, year_exams: str, delete: bool = False):
...
+                                current_exams = new_exam.get_session("prima") + new_exam.get_session("seconda") + \
+                                    new_exam.get_session("terza") + new_exam.get_session("straordinaria")
+                                if (session == "straordinaria" or cells.index(cell) == 5) \
+                                        and cell_clean_text not in current_exams\
+                                        and cell_clean_text != "":
+                                    new_exam.append_session("straordinaria", cell_clean_text)
+                                elif session in ["prima", "seconda", "terza"] \
+                                        and cell_clean_text not in current_exams \
+                                        and cell_clean_text != "":
+                                    new_exam.append_session(session, cell_clean_text)
```
Comment (Helias) [inline module/data/exam.py:193]: same as above

File: module/callback_handlers.py [hunk 5]
```diff
@@ -9,11 +9,12 @@
 # Needed to correctly run functions using globals()
 from module.utils.multi_lang_utils import get_on_demand_text
 
-from module.commands.esami import *  # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.lezioni import *  # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.help import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.drive_contribute import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.aulario import * # pylint: disable=wildcard-import,unused-wildcard-import
+import logging  # pylint: disable=wildcard-import,unused-wildcard-import
```
Comment (Helias) [inline module/callback_handlers.py]: remove `# pylint: disable=wildcard-import,unused-wildcard-import` where you can

File: module/data/exam.py [hunk 6]
```diff
@@ -81,6 +81,16 @@ def append_session(self, session_name: str, to_append: str):
             # pylint: disable=unnecessary-dunder-call
             self.__getattribute__(session_name).append(to_append)
 
+    @classmethod
+    def append_multiple_sessions(cls, cells, exam, session):
+        for cell in cells:
+            cell_clean_text = cell.text.replace('\xa0', '').replace('\n', '').replace('DMI', '')
+            current_exams = [string.replace('DMI', '') for string in exam.get_session("prima") +
+                             exam.get_session("seconda") + exam.get_session("terza") +
+                             exam.get_session("straordinaria")]
```
Comment (Helias) [inline module/data/exam.py:90]: this is a bit long, isn't it?
```suggestion
            exam_sessions = exam.get_session("prima") + exam.get_session("seconda") + exam.get_session("terza") + exam.get_session("straordinaria")
            current_exams = [string.replace('DMI', '') for string in exam_sessions]
```

File: module/callback_handlers.py [hunk 7]
```diff
@@ -9,9 +9,12 @@
 # Needed to correctly run functions using globals()
 from module.utils.multi_lang_utils import get_on_demand_text
 
-from module.commands.help import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.drive_contribute import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.aulario import * # pylint: disable=wildcard-import,unused-wildcard-import
+import logging  # pylint: disable=unused-import
```
Comment (Helias) [inline module/callback_handlers.py:12]: remomve pylint warning when you can

File: module/callback_handlers.py (hunk 5 above)
Comment (scraccio) [reply module/callback_handlers.py]: If I remove those, pylint won't give a score of 10 and the deploy will fail

are you suggesting to replace this solution with a smoother one, or to straight up remove these imports?

**Review by Helias — APPROVED** [review]

**Pull request merged**